tests/e2escenarios/e2e_images_test.go

```diff
@@ -36,21 +36,17 @@ var _ = Describe("odo supported images e2e tests", func() {
 	// Also verify the flow of odo commands with respect to supported images only.
 	verifySupportedImage := func(image, srcType, cmpType, project, appName, context string) {
 
+		cmpName := srcType + "-app"
 		// create the component
 		helper.CopyExample(filepath.Join("source", srcType), commonVar.Context)
-		helper.CmdShouldPass("odo", "create", "--s2i", cmpType, srcType+"-app", "--project", project, "--context", context, "--app", appName)
-
-		helper.CmdShouldPass("odo", "config", "set", "minmemory", "400Mi", "--context", context)
-		helper.CmdShouldPass("odo", "config", "set", "maxmemory", "700Mi", "--context", context)
+		helper.CmdShouldPass("odo", "create", "--s2i", cmpType, cmpName, "--project", project, "--context", context, "--app", appName)
 
 		// push component and validate
 		helper.CmdShouldPass("odo", "push", "--context", context)
 		cmpList := helper.CmdShouldPass("odo", "list", "--context", context)
 		Expect(cmpList).To(ContainSubstring(srcType + "-app"))
 
-		// create a url
-		helper.CmdShouldPass("odo", "url", "create", "--port", "8080", "--context", context)
-		helper.CmdShouldPass("odo", "push", "--context", context)
```

Comment:
We should have a second push in the testing scenario. 
In past, we had a situation where the second `odo push` broke a component. 
This scenario should have a second push in it. For example, you should change something using `odo config` and do `odo push`  again and verify that the application is still running even after the second push.

Reply:
added